### DISCUSSION TITLE
Collapsible panel with toolbar won't collapse

### DISCUSSION THREAD

**Discussion by magdicom** [laravel/nova-issues#4750] [General]
- Laravel Version: 9.26.1
- Nova Version: 4.13.0
- PHP Version: 8.1.3
- Database Driver & Version: Mysql 8.0.28
- Operating System and Version: Homestead, Windows 11
- Browser type and version: Chrome 104
- Reproduction Repository: ..

### Description:
When you define a panel to display toolbar and collapsible only the toolbar will be displayed and the toggle button (arrow after panel title) won't show, hence you can't show/hide the panel.

Toolbar will only affect the toggle feature, if the panel was closed/hidden from before, there is no way to open it until you make the toolbar go away from the `fields()` function

### Detailed steps to reproduce the issue on a fresh Nova installation:

```
public function fields(NovaRequest $request)
{
        return [
            Panel::make('More Fields', [
                // Fields
            ])
            ->collapsable()
            ->withToolbar(),
        ];
}

```

**Comment by crynobone** [comment]
AFAIK at the moment `withToolbar()` is an undocumented feature specifically built for Relations Panel. Can you provide a use-case where this can be used other than Relations Panel?
